components/org.wso2.carbon.identity.oauth.dcr/src/main/java/org/wso2/carbon/identity/oauth/dcr/bean/Application.java

```diff
@@ -26,7 +26,7 @@
     private static final long serialVersionUID = -4515815791420125411L;
 
     private String client_name = null;
-    private String client_key = null;
+    private String client_id = null;
```

Comment:
Shall we get rid of the '_'s and use the camel case notation

Reply:
Those variables are created in this format as this object is used in a response from the endpoint.